index.html

```diff
@@ -146,25 +245,32 @@ <h5>About me</h5>
         </div>
     </section>
     <footer>
-        <h1>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
-        <form action="https://formspree.io/f/mlevqbqb" method="post"> 
-            <ul>
-                <li>
-                    <input name="user_name" type="text" id="name" maxlength="30" value="Full name" required />
-                </li>
-                <li>
-                    <input name="user_mail" type="email" id="email" value="Email" required />
-                </li>
-                <li>
-                    <textarea name="user_msg" id="message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
-                </li>
-                <li>
-                    <button class="submit-btn" type="submit">Get in touch</button>
-                </li>
-            </ul>
-        </form>
-        <hr class="solid">
-        <div class="social-media-icon">
+        <div class="form-form">
+            <div class="interest">
+                <h1>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            </div>
```

Comment:
- [OPTIONAL]  :thinking: I think it would be better to use the same font-family, align the text to the center as the figma to make them it match
- [OPTIONAL] Please the change the h1 into another hx since we already a h1 on this webpage we should avoid it reuse for sementic purpose.

__Please note that these requirement is crucial for the project, it's optional here only because the previous reviewers didn't catch it__

- Your design:
![YourTitle](https://user-images.githubusercontent.com/97834160/206587183-8b079151-b445-4c80-b991-6f0baecfa8df.png)

- Figma:
![FigTitle](https://user-images.githubusercontent.com/97834160/206587240-c8bbf73c-d2b7-4412-8a97-9f7f5b62e304.png)